{"uuid":"2517c327-24f6-4356-86f4-49719f4a064a","historyId":"e7acb588d1ab8cf06cb3357bf9a390a3","fullName":"testCasesByW3cTouchActions.LoginTests.successfulLogin","labels":[{"name":"package","value":"testCasesByW3cTouchActions.LoginTests"},{"name":"testClass","value":"testCasesByW3cTouchActions.LoginTests"},{"name":"testMethod","value":"successfulLogin"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"SwagLabsAppByAppium2"},{"name":"subSuite","value":"testCasesByW3cTouchActions.LoginTests"},{"name":"host","value":"Yehia-Metwally"},{"name":"thread","value":"17652@Yehia-Metwally.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"epic","value":"SwagLabs Android App"},{"name":"feature","value":"User Login"},{"name":"story","value":"Verify User Login"}],"links":[],"name":"successfulLogin","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"This is the Error\n\texpected [yyyy] but found [PRODUCTS]","trace":"java.lang.AssertionError: This is the Error\n\texpected [yyyy] but found [PRODUCTS]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat testCasesByW3cTouchActions.LoginTests.successfulLogin(LoginTests.java:64)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[{"name":"Enter Username","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"username","value":"standard_user"}],"start":1726274967402,"stop":1726274970061},{"name":"Enter Password","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"password","value":"secret_sauce"}],"start":1726274970065,"stop":1726274972727},{"name":"Click on Submit Button","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1726274972728,"stop":1726274974679},{"name":"test","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1726274974679,"stop":1726274977815}],"attachments":[],"parameters":[],"start":1726274966687,"stop":1726274979345}